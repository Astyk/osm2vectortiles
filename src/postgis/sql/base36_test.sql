CREATE EXTENSION base36;
SELECT point_to_tiles(1, 1);
