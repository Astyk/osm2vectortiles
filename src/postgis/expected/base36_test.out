CREATE EXTENSION base36;
SELECT retcomposite(1, 1);
 retcomposite 
--------------
 (1,2,3)
(1 row)

